README.md

```diff
@@ -1,10 +1,10 @@
-[![Build Status](https://travis-ci.com/w3c/aria-practices.svg?branch=master)](https://travis-ci.com/w3c/aria-practices)
+[![Build Status](https://travis-ci.com/w3c/aria-practices.svg?branch=main)](https://travis-ci.com/w3c/aria-practices)
```

Comment:
This should probably just be deleted, since Travis isn't used anymore

Reply:
Updated PR incoming. :)